stdlib/2and3/optparse.pyi

```diff
@@ -0,0 +1,226 @@
+# Generated by pytype, with only minor tweaks. Might be incomplete.
+import sys
+from typing import Any, Optional, List, Callable, Tuple, Dict, Iterable, Union, Mapping, IO
+
+# See https://groups.google.com/forum/#!topic/python-ideas/gA1gdj3RZ5g
+if sys.version_info >= (3,):
+    Text = str
+else:
+    Text = Union[str, unicode]
```

Comment:
If you do have to define your own type, I'd prefer if you gave it a new name. There's already a `typing.Text`, so this is confusing. `OptText`? `_Text`? `AnyText`?